nextstrain/cli/volume.py

```diff
@@ -37,7 +36,4 @@ def __call__(self, parser, namespace, values, option_strings = None):
             new_volume = NamedVolume(volume_name, Path(values)) if values else None
             setattr(namespace, "volumes", [*volumes, new_volume])
 
-            # Allow the new volume to be found by name on the opts object
-            setattr(namespace, volume_name.replace('/', '_'), new_volume)
```

Comment:
Curious - what was the purpose of `replace('/', '_')`? I see that `volume_name` takes on values `augur`, `auspice`, `fauna`, `sacra`, and previously `build` - none of which were affected by this replacement.

Reply:
Prior to a0f3ad197f0d3946c090ed51b0230ea30ac68776 there [was](https://github.com/nextstrain/cli/commit/a0f3ad197f0d3946c090ed51b0230ea30ac68776#diff-8846c03699df577b0083c116e35a2dcde0f3093f6e58606f9584de266f77cdc7L126-R138) a `store_volume("auspice/data")` call for `nextstrain view`.